nautobot/extras/templates/extras/customfield.html

```diff
@@ -0,0 +1,94 @@
+                        <td><span>{{ object.type | capfirst }}</span></td>
+                    </tr>
+                    <tr>
+                        <td>Weight</td>
+                        <td><span>{{ object.weight }}</span></td>
+                    </tr>
+                    <tr>
+                        <td>Required</td>
+                        <td>
+                            {% if object.required %}
+                                <span class="text-success"><i class="mdi mdi-check-bold"></i></span>
+                            {% else %}
+                                <span class="text-danger"><i class="mdi mdi-close"></i></span>
+                            {% endif %}
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>Description</td>
+                        <td><span>{{ object.description }}</span></td>
+                    </tr>
+                    <tr>
+                        <td>Default</td>
+                        <td><span>{{ object.default }}</span></td>
+                    </tr>
+                    <tr>
+                        <td>Filter Logic</td>
+                        <td><span>{{ object.filter_logic | capfirst }}</span></td>
+                    </tr>
+                </table>
+            </div>
+{% endblock content_left_page %}
+
+{% block content_right_page %}
+            <div class="panel panel-default">
+                <div class="panel-heading">
+                    <strong>Validation Rules</strong>
+                </div>
+                <table class="table table-hover panel-body attr-table">
+                    <tr>
+                        <td>Minimum Value</td>
+                        <td><span>{{ object.validation_minimum }}</span></td>
+                    </tr>
+                    <tr>
+                        <td>Maximum Value Type</td>
+                        <td><span>{{ object.validation_maximum }}</span></td>
+                    </tr>
+                    <tr>
+                        <td>Validation Regex</td>
+                        <td>
+                            {% if object.validation_regex %}
+                                <code>{{ object.validation_regex }}</code>
+                            {% else %}
+                                <span>None</span>
+                            {% endif %}
+                        </td>
+                    </tr>
+                </table>
+            </div>
+            <div class="panel panel-default">
+                <div class="panel-heading">
+                    <strong>Assignment</strong>
+                </div>
+                <table class="table table-hover panel-body attr-table">
+                    <tr>
+                        <td>Content Types</td>
+                        <td>
+                            {% for ct in object.content_types.all %}
+                                {{ ct }}<br>
+                            {% endfor %}
```

Comment:
Possibly use a `<ul>` and `<li>` for proper list semantics, instead of `<br>`? Though I see that this is how the `status` detail page currently renders its content-types - not sure if better to be correct or consistent here. :-)